src/main/java/fi/helsinki/cs/tmc/core/communication/UrlCommunicator.java

```diff
@@ -149,11 +149,10 @@ private MultipartEntityBuilder addParamsToRequest(
      * Tries to make GET-request to specific url.
      *
      * @param url URL to make request to
-     * @param params Any amount of parameters for the request. params[0] is always username:password
      * @return A Result-object with some data and a state of success or fail
```

Comment:
Maybe like:
Tries to make a GET-request and returns object representing the result.